### PR TITLE
fix the image gallery in index.js and tag.js

### PR DESCRIPTION
**Pull request by philipkobernik**

### CODE REVIEW

File: components/project-preview.js [hunk 1]
```diff
@@ -10,26 +10,34 @@ export default function ProjectPreview({
   excerpt,
   author,
   authorSlug,
+  tags,
   slug,
 }) {
   return (
-    <div class="flex">
-      <div className="w-1/3 pr-8">
+    <div class="grid grid-cols-3">
```
Comment (philipkobernik) [inline components/project-preview.js]: great opportunity to change `class` to `className`

File: components/project-preview.js [hunk 2]
```diff
@@ -10,26 +10,34 @@ export default function ProjectPreview({
...
+      <div className="col-start-1 col-end-2 pr-8">
         <CoverImage
           slug={slug}
           title={title}
           responsiveImage={coverImage.responsiveImage}
         />
       </div>
-        <div class="flex-wrap">
+
+      <div class="col-start-2 col-end-4">
```
Comment (philipkobernik) [inline components/project-preview.js]: `className`

File: components/project-preview.js [hunk 3]
```diff
@@ -10,26 +10,34 @@ export default function ProjectPreview({
...
         <h3 className="w-full text-3xl mb-4 leading-snug">
           <Link as={`/projects/${slug}`} href="/projects/[slug]">
             <a className="hover:underline">{title}</a>
           </Link>
         </h3>
+
         <p className="w-full text-lg leading-relaxed mb-4">{excerpt}</p>
+
+        <div className="mb-6 text-lg">
+          { tags.map(t => (<Link as={`/tags/${t.slug}`} href="/tags/[slug]"><a className="text-orange hover:underline">{t.name} </a></Link>))}
```
Comment (philipkobernik) [inline components/project-preview.js]: whitespace can help readability here
```
{
  tags.map(t => (
    <Link as={`/tags/${t.slug}`} href="/tags/[slug]">
      <a className="text-orange hover:underline">{t.name} </a>
    </Link>
  ))
}
```

also, whenever you render a list of things with map, you'll want to include a `key` prop with a unique value. This allows react to manage lists of things in a performant way.

```
{
  tags.map(t => (
    <Link as={`/tags/${t.slug}`} href="/tags/[slug]" key={t.slug}>
      <a className="text-orange hover:underline">{t.name} </a>
    </Link>
  ))
}
```

Comment (sambourgault) [reply components/project-preview.js]: okai!

**Pull request merged**